sky/packages/sky/lib/src/rendering/box.dart

```diff
@@ -259,17 +291,19 @@ abstract class RenderBox extends RenderObject {
     _debugDoingBaseline = value;
     return true;
   }
-  // getDistanceToBaseline() returns the distance from the
-  // y-coordinate of the position of the box to the y-coordinate of
-  // the first given baseline in the box's contents. This is used by
-  // certain layout models to align adjacent boxes on a common
-  // baseline, regardless of padding, font size differences, etc. If
-  // there is no baseline, and the 'onlyReal' argument was not set to
-  // true, then it returns the distance from the y-coordinate of the
-  // position of the box to the y-coordinate of the bottom of the box,
-  // i.e., the height of the box. Only call this after layout has been
-  // performed. You are only allowed to call this from the parent of
-  // this node during that parent's performLayout() or paint().
+
+  /// Returns the distance from the y-coordinate of the position of the box to
+  /// the y-coordinate of the first given baseline in the box's contents.
+  ///
+  /// This is used by certain layout models to align adjacent boxes on a common
+  /// baseline, regardless of padding, font size differences, etc. If there is
+  /// no baseline, and the 'onlyReal' argument was not set to true, then it
+  /// returns the distance from the y-coordinate of the position of the box to
+  /// the y-coordinate of the bottom of the box, i.e., the height of the box.
+  ///
+  /// Only call this function after layout has been performed. You are only
```

Comment:
"Only call this function on a box after its layout() method has been called." maybe better.